cloudflared/rootfs/etc/cont-init.d/10-cloudflared-config.sh

```diff
@@ -352,6 +360,112 @@ hasCustomConfig() {
     bashio::exit.nok "No custom config found: ${data_path}/config.yml please create custom config file or remove 'custom_config' option"
 }
 
+# ------------------------------------------------------------------------------
+# Create route for local IPs if Warp is enabled
+# ------------------------------------------------------------------------------
+createRoute() {
+    if ! bashio::config.has_value 'warp_enable' || bashio::config.false 'warp_enable' ; then
+        return "${__BASHIO_EXIT_OK}"
+    fi
+
+    # Remove already linked routes 
+    bashio::log.info "Removing already configured routes for tunnel ${tunnel_name}"
+    # Get routes linked to tunnel id
+    existing_tunnel_routes=$(cloudflared --origincert="${data_path}/cert.pem" \
```

Comment:
I think we can take this deletion part and extract it to it's own method and then share it with "warp_reset". Does that make sense?